### DISCUSSION TITLE
A more secure approach? (Py or Exe) [question]

### DISCUSSION THREAD

**Discussion by eViruxe** [dashingsoft/pyarmor#1189] [General]
Hello there,
what would be a more secure approach. Just obfuscating a python script using PyArmor 8.0 (Pro) or converting the python script into exe using PyInstaller then packing the exe file using PyArmor?

**Comment by jondy** [comment]
In current pyarmor 8.1.x, they're almost same.

In next month, some new features may be added to improve security of exe
1. obfuscating all the `.pyc` of system package in the bundle
2. binding obfuscated script to bundle loader, so that the obfuscated scripts couldn't run by any other Python interpreter

**Reply by eViruxe** [reply]
I'm currently using `pyarmor gen -e 2023-06-15 --enable-bcc --enable-jit --mix-str --assert-call --assert-import --private --pack dist\Script.exe Script.py`
Which approach would you recommend currently and next month?

**Comment by jondy** [comment]
Please check the latest documentation and releases in the GitHub.

Now it's not documented yet.

**Reply by eViruxe** [reply]
Understandable. Currently which approach would you recommend? Py Or Exe ?
